backend/app/member_handlers.go

```diff
@@ -0,0 +1,61 @@
+package app
+
+import (
+	"strconv"
+
+	"github.com/codescalersinternships/Flyspray/models"
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+)
+
+func (a *App) CreateNewMember(c *gin.Context) (interface{}, Response) {
+	var member models.Member
+	if err := c.ShouldBindJSON(&member); err != nil {
```

Comment:
create a `createMemberInput` struct for data retrieved form user, the model data is different than user input